APIComparer.Contracts/CompareNugetPackage.cs

```diff
@@ -2,21 +2,18 @@
 namespace APIComparer.Contracts
 {
     using NServiceBus;
-    using NuGet;
 
     public class CompareNugetPackage : ICommand
     {
-        public CompareNugetPackage(string packageId, SemanticVersion leftVersion, SemanticVersion rightVersion, string target)
+        public CompareNugetPackage(string packageId, string leftVersion, string rightVersion)
```

Comment:
What is the reason to change this? I thought we previously agree on using SemanticVersion on the contract